CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 6.4.0
+
+- Updated the Mastodon version to v4.4.0
```

Comment:
We should tell people to read the 4.4.0 release notes: https://github.com/mastodon/mastodon/releases/tag/v4.4.0

Especially:
- redis & postgres minimum version bump
- dropped support for redis namespaces
- no-downtime updates from versions before 4.3.0 not supported
- the search mappings need to be updated manually (run the mentioned `tootctl` command from a pod) after the update, if ES is used
- values should be changed to ensure that the `fasp` queue is processed by Sidekiq